brent/graph.py

```diff
@@ -269,6 +274,33 @@ def add_edge(self, source, sink) -> 'DAG':
         logging.debug(f"created connection {source} -> {sink}")
         return self
 
+    def remove_edge(self, source, sink) -> 'DAG':
+        """
+        Removes an existing edge from the graph.
+
+        ## Input
+
+        - **source**: Name of a node in the graph
+        - **sink**: Name of a node in the graph
+
+        ## Example
+
+        ```
+        from brent import DAG
+        from brent.common import make_fake_df
+
+        (DAG(dataframe=make_fake_df(4))
+            .add_edge("a", "b")
+            .remove_edge("a", "b"))
+        ```
+        """
+        try:
```

Comment:
i might prefer a check (and a test). if the DAG is cached then this action should no longer be allowed.

Reply:
i'm not super sure what is great behavior with regards to that cacheing tho. might need to discuss this in person.

Reply:
Good point, I'll add a test. Let's discuss caching in person indeed